### PR TITLE
Improve page on DAOs

### PR DESCRIPTION
**Pull request by scottfits**
Add or improve sections for:
- DAO governance
- DAO laws
- DAO examples

### CODE REVIEW

File: src/content/dao/index.md [hunk 1]
```diff
@@ -79,25 +129,6 @@ _Typically used for decentralized development and governance of protocols and da
 
 [DXdao](https://DXdao.eth.link) – DXdao is a global sovereign collective building and governing decentralized protocols and applications since 2019. It leverages reputation-based governance and holographic consensus to coordinate and manage funds, meaning no one can buy their way into influencing its future.
 
-## How do DAOs work? {#how-daos-work}
```
Comment (minimalsm) [inline src/content/dao/index.md:82]: Why we removing this section? (Has it been moved? I don't see elsewhere)

**Review by minimalsm — COMMENTED** [review]
Hey @scottfits, thanks for the PR here. Will try to get to reviewing this very soon ❤️

File: src/content/dao/index.md (hunk 1 above)
Comment (scottfits) [reply src/content/dao/index.md:82]: I moved it higher up on the page, above the DAO membership section (this section is way more important than the DAO membership one)

image

**Comment by scottfits** [comment]
@minimalsm sorry to bother you. i think this PR really improves the page. Here is an overview of the improvements: and some screenshots:


### 1. This is the main new section in this PR that explains DAO governance methods

> image

### 2. Make the definition of DAO better and more specific

> image


### 3. We move this section lower on the page since it's an implementation detail and most DAOs are using token-based membership

> image


### 4. We replace _freelancer network_ with _collective ownership_ since that is becoming a more compelling mainstream use case for DAOs.

> image

### 5. Fix grammar and sentence fragments

> image

**Review by minimalsm — APPROVED** [review]
Sorry for the delay here @scottfits,

I think this is great! :-) thanks again.

**Comment by minimalsm** [comment]
@all-contributors please add @scottfits for docs

**Pull request merged**